01_01_Basic_Rules.txt

```diff
@@ -625,4 +642,4 @@ out a receiver, you may carry (at max) one additional receiver in order to swap
 kits on the go. You must pass 2 Weapons use / knowledge [Weapons] checks in 
 order to successfully convert your weapon on the go. DC 60. Failure on one check 
 instantly jams the weapon on first shot. Failure on both checks requires an 
-additional check.
\ No newline at end of file
+additional check.

```

Comment:
EOF changed from windows/linux standard. Not sure why. Shouldn't break anything. Don't really care.